src/uwtools/tests/utils/test_api.py

```diff
@@ -105,16 +104,14 @@ def test_str2path_convert():
 @mark.parametrize("hours", [0, 24, 168])
 def test__execute(execute_kwargs, hours, tmp_path):
     graph_file = tmp_path / "g.dot"
-    with patch.object(sys.modules[__name__], "TestDriverWCL", wraps=TestDriverWCL) as cd:
```

Comment:
No need to mock this, and doing so causes problems with the `getfullargspec()` call deeper in the call stack.